src/latte/core.clj

```diff
@@ -5,6 +5,8 @@
   Users (as opposed to developpers) of the framework should
   mostly depend on this namespace."
 
+  (:refer-clojure :exclude [apply])
```

Comment:
I don't think it's good to shadow such an important function as apply,
let's call it with some other name (more about that below)